File: C#/Sources/MSTEFramework/MSTEException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MSTEFramework {
    public class MSTEException : Exception {

	    public MSTEException() : base() {
	    }
	    public MSTEException(String s) : base(s) {
		    
	    }
    }
}

File: Java/src/com/openmicrostep/mste/MSTEException.java
package com.openmicrostep.mste;

public class MSTEException extends Exception {

    public MSTEException() {
        super();
    }

    public MSTEException(String s) {
        super(s);
    }

    public MSTEException(String s, Exception e) {
        super(s, e);
    }
}
